src/Settings.php

```diff
@@ -44,38 +44,47 @@ static function () {
      *
      * @return string
      */
-    protected function renderSettingsPage(): string
+    protected function renderSettingsPage(): string //phpcs:disable Inpsyde.CodeQuality.FunctionLength.TooLong
     {
         ob_start(); ?>
         <div class="wrap">
-            <h1>eCurring Settings</h1>
+            <h1><?php
+                echo esc_html_x('eCurring Settings', 'Settings page title', 'woo-ecurring');
```

Comment:
Is it eCurring settings? shouldn't be Mollie Subscriptions Settings?

Reply:
Oh, this is what you meant. I created a new issue for that https://inpsyde.atlassian.net/jira/software/c/projects/ECUR/issues/ECUR-79.

Reply:
Anyway, fixed. I removed the `eCurring` part and left just `Settings`. Since this text is displayed under the plugin title, no need to duplicate it.

Reply:
Ouch, just realized I renamed it in another place. There it had to be renamed too, though. Now it's ok in both places.